networkx/algorithms/bipartite/matching.py

```diff
@@ -475,18 +475,18 @@ def to_vertex_cover(G, matching, top_nodes=None):
 
 
 def minimum_weight_full_matching(G, top_nodes=None, weight='weight'):
-    r"""Returns the minimum weight full matching of the bipartite graph `G`.
+    r"""Returns a minimum weight full matching of the bipartite graph `G`.
```

Comment:
Why the change here? I'm not familiar with the matching algorithms so forgive me if this is an obvious question. Are there sometimes multiple minimum-weight full-matchings for bipartite graphs?

Reply:
@rossbar There can be many minimum-weight full-matchings for bipartite graphs.  A matching is an independent edge set of the graph (i.e., no two edges share a common node).  So, e.g., if you have a balanced, complete bipartite graph (each part has an equal # of nodes with no edges in each part and all possible edges between the parts) whose edges all have unit weight, then there are a large number of minimum weight matchings.

Reply:
Yeah, that's exactly it. Say, for instance, that the partitions are `{1, 2}` and `{3, 4}`, with edges `(1, 3)`, `(1, 4)`, `(2, 3)`, `(2, 4)`, all with equal weights. Then there are two full matchings with total weight 2: `[(1, 3), (2, 4)]` and `[(1, 4), (2, 3)]`.

Really, some of the other methods in this module should be changed as well to be completely precise.

Reply:
Thanks all for the clarification!